app/controllers/users_controller.rb

```diff
@@ -0,0 +1,53 @@
+class UsersController < ApplicationController
+  skip_before_action :require_login, only: :index
+
+  def index
+    @users = User.all
+    render layout: "application"
+  end
+
+  def show
+    # creates a unique list of the logged in user's order statuses
+    @order_statuses = @logged_in_user.order_items.map { |order_item|
```

Comment:
Another good opportunity for a model method - either on the user or the order item